pootle/tools/createpootlepot

```diff
@@ -17,7 +17,7 @@ POOTLE_JS_POT="$POT_DIR/pootle_js.pot"
 
 # Header settings
 PACKAGE="Pootle"
-VERSION=`egrep "^ver" $SRC_DIR/__version__.py | sed "s,.*(\(.*\)),\1,g" | sed "s,[ ],,g" | sed "s/,/./g"`
+VERSION=$(python -c "from pootle.core.utils.version import *; print(get_main_version())")
```

Comment:
A thought comes to mind, if you want to use it like this you could create an `if __name__ == "__main__"` block in the file which prints the version, and run it with `python -mpootle.core.utils.version`. Cleaner, and doesn't break once we start using relative imports more.

Otherwise, gtm.